### ISSUE TITLE
Feature Enhancement: Expanded Most Used Sermon Series / Speakers

### ISSUE THREAD

**Issue by davidshq** [jtsternberg/GC-Sermons#20]
It would be great if these two on a sermon edit page would show a few of the most used options, especially for speakers.

**Comment by jtsternberg** [comment]
You mean like this? 
![](http://b.ustin.co/1jQZr+)

**Comment by davidshq** [comment]
Yes, except for it to be expanded from the start...though perhaps only a few and then a more option...for me, by the time I click on most used, I could almost have written the name in again.

**Comment by davidshq** [comment]
Found this article on StackOverflow on how to accomplish this:
http://wordpress.stackexchange.com/questions/45149/how-to-have-the-most-used-tags-taxonomy-always-expanded

This article is similar but shows how to show all tags by default...Not what I'd want, but still interesting:
http://wordpress.stackexchange.com/questions/174593/show-all-post-tags-on-post-edit-screen-sidebox
